### PR TITLE
Remove ref usage and upgrade to sb 6

### PR DESCRIPTION
**Pull request by scaredcat**
### Changes
- Update to storybook 6 using [this guide](https://medium.com/storybookjs/storybook-6-migration-guide-200346241bb5)
- Make use of the new [component story format](https://medium.com/storybookjs/component-story-format-66f4c32366df)
- Remove unnecessary use of refs

Fixes: #9

### CODE REVIEW

File: src/base-folder.js [hunk 1]
```diff
@@ -39,23 +39,13 @@ class BaseFolder extends React.Component {
...
+      const currentName = element.value
+      element.setSelectionRange(0, currentName.length)
+      element.focus()
     }
   }
-  selectAllNewName = () => {
-    window.requestAnimationFrame(() => {
-      const currentName = this.newNameRef.value
-      this.newNameRef.setSelectionRange(0, currentName.length)
-      this.newNameRef.focus()
```
Comment (mwfister) [inline src/base-folder.js:56]: There's [one final reference](https://github.com/uptick/react-keyed-file-browser/blob/85cc881e11ec6e76cdb8a0c5be0b2900745d40d4/src/base-folder.js#L74) to this `ref`, which will prevent renaming folders.

**Review by mwfister — CHANGES_REQUESTED** [review]
Thanks for the changes @scaredcat! All in all looks good, just needs the folder name change handler updated

File: src/base-folder.js (hunk 1 above)
Comment (scaredcat) [reply src/base-folder.js:56]: oops. I've removed that `newNameRef` as well.

**Review by mwfister — APPROVED** [review]

**Comment by mwfister** [comment]
Looks good to merge. We'll look into creating a new release soon™, ideally once the other active PRs are resolved.

**Pull request merged**